source/CommandLine/main.cpp

```diff
@@ -45,44 +45,44 @@ int VIREO_MAIN(int argc, const char * argv[])
     }
 
     if (argc >= 2) {
+        gShells._pRootShell = TypeManager::New(null);
```

Comment:
I see your point about the change in this file (main.cpp). However, I tend to think this need not be done as part of this PR. We should create a Vireo release without this change and with only the other fixes. This may be done separately as a commit in a different PR and with adding some tests with the workflow of passing more than one VIA file as input.

Reply:
This to me is like submitting test via files together with the change that fixes them.
With this change, it's possible to reproduce buggy behavior associated with this fix using the native driver by running WildCardArgumentSubVICalls.vi twice, and without it, it is not.
I don't think it's important whether or not it is rolled into this Vireo release because main.cpp doesn't affect the JS build at all.   So I'm pushing back on this, but if you feel strongly, I can pull it out.

Reply:
I did a quick sanity test on this locally now. With this change, it looks like if two VIA files are passed as input to Vireo native, output from the first VIA file goes to the console twice and output from second VIA file does not go to the console. If we can fix this (and optionally/preferably add a small test on this console behavior), let's check this in.

We are using Vireo Native in the NXG side of CI tests, so we need to make sure that won't be broke with Vireo updates.

Reply:
Ok, please try again, I think I fixed it.   The user scope still needs to be instantiated per via file or the global namespace might have conflicts.